apps/android/app/src/main/res/values-ru/strings.xml

```diff
@@ -152,6 +153,8 @@
     <string name="icon_descr_simplex_team"><xliff:g id="appName">SimpleX</xliff:g> команда</string>
     <string name="image_descr_simplex_logo"><xliff:g id="appName">SimpleX</xliff:g> логотип</string>
     <string name="icon_descr_email">Email</string>
+    <string name="icon_checkmark_action">TODO</string>
+    <string name="icon_paste_from_clipboard">TODO</string>
```

Comment:
```suggestion
    <string name="icon_paste_from_clipboard">Вставить из буфера обмена</string>
```

Reply:
I don't see where this and `icon_checkmark_action`, were they deleted from code but not from translations?